packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/utils/getOvirtProviderWebUILink.ts

```diff
@@ -16,7 +16,7 @@ export const getOvirtProviderWebUILink = (provider: V1beta1Provider): string =>
   }
 
   // Calculate link using API URL
-  const uiLinkRegexp = new RegExp('(?<=ovirt-engine)/api(/)?$', 'g');
+  const uiLinkRegexp = new RegExp('(?<=ovirt-engine)/api(/)*$', 'g');
```

Comment:
that does not look right, can you what am i missing ?

Reply:
This is for supporting extra slashes in the end of the given URL, E.g.:
if URL is set to `https://rhev.redhat.com/ovirt-engine/api/////` which is a valid path and we won't block it, then we will still auto calc the ui link properly.
It's just a nice enhancement.

Reply:
if a user want to do garbage in, they will get grabage out :-) i am ok with supporting one `/` but `/////` is grabage, please don't support it here